scipy/optimize/_linprog_util.py

```diff
@@ -586,6 +587,17 @@ def where(A):
         where = np.where
         vstack = np.vstack
 
+    # upper bounds >= lower bounds
```

Comment:
I'm forgetting - is this being moved from somewhere else? But I don't see it deleted anywhere else in the changes.

Also, shouldn't the comparison be string inequality? Upper and lower bounds equal is just an equality constraint. The variable is fixed and can be removed from the problem (a presolve operation).

Reply:
This check was part of `_clean_inputs()`, it was removed there since I'd like to have all infeasibility tests in one location. And you are right: there must not be an equal sign.

Reply:
On second thought: why did this pass the tests? I'll take a closer look at this.

Reply:
There were no tests which explicitly checked fixed bounds. I have added one.